lib/tasks/invite_mentors.rake

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+namespace :invites do
+  desc "Invite mentor users"
+  task send_mentor_invites: :environment do
+    puts("Sending mentor invites...")
+
+    unsent_emails = InviteEmailMentor.where(sent_at: nil)
+    unsent_emails.each do
+      email.send!(force_send: true)
+    rescue Notifications::Client::RateLimitError
+      sleep(1)
```

Comment:
sleeps the equivalent of setInterval in JS? Why's it needed here? Is it incase we're sending them to quickly after each other and this just adds a delay to allow the task to continue without failing completely? If that's right, what's the rate limit?

Reply:
360 per second by the looks of it

Reply:
Yeah, that is what it does. I think 1 second is enough, I stole the idea from R&P who have a lot more experience sending vast amounts of emails.